apps/condo/domains/billing/schema/RegisterBillingReceiptsService.test.js

```diff
@@ -554,4 +558,59 @@ describe('RegisterBillingReceiptsService', () => {
         })
     })
 
+    describe('Cache', () => {
+        const REDIS_CLIENT_NAME = 'register-billing-receipt'
+        const redisClient = getRedisClient(REDIS_CLIENT_NAME, 'cache')
```

Comment:
You can't use redis Client inside test.js files!

You should use only API calls inside tests. If you want some redis checks you need to use .spec.js file